sdk/typescript/src/types/common.ts

```diff
@@ -32,22 +35,52 @@ export function isValidTransactionDigest(value: string): value is TransactionDig
 // which uses the Move account address length
 // https://github.com/move-language/move/blob/67ec40dc50c66c34fd73512fcc412f3b68d67235/language/move-core/types/src/account_address.rs#L23 .
 
-const SUI_ADDRESS_LENGTH = 20;
+export const SUI_ADDRESS_LENGTH = 20;
 export function isValidSuiAddress(value: string): value is SuiAddress {
-  return isHex(value) &&
-    getHexByteLength(value) === SUI_ADDRESS_LENGTH;
+  return isHex(value) && getHexByteLength(value) === SUI_ADDRESS_LENGTH;
 }
 
 export function isValidSuiObjectId(value: string): boolean {
   return isValidSuiAddress(value);
 }
 
+/**
+ * Perform the following operations:
+ * 1. Make the address lower case
+ * 2. Prepend `0x` if the string does not start with `0x`.
+ * 3. Add more zeros if the length of the address(excluding `0x`) is less than `SUI_ADDRESS_LENGTH`
+ *
+ * WARNING: if the address value itself starts with `0x`, e.g., `0x0x`, the default behavior
+ * is to treat the first `0x` not as part of the address. The default behavior can be overridden by
+ * setting `forceAdd0x` to true
+ *
+ */
+export function normalizeSuiAddress(
```

Comment:
Can you export these from the top-level module as well? I ran into a case where I need them and think they'd be generally useful.

Reply:
Correct me if I am wrong, but they should already be exported from the top-level module